README.md

```diff
@@ -26,6 +26,10 @@ To read more about interchain applications, how the protocol works, and how to i
 
 ## Working on Hyperlane
 
+### Prerequisites
```

Comment:
Great point to add a prerequisites step, with this I'd now move the other dependency install steps (`Foundry`, `Node`, `Workspaces`) under `Prerequisites` by changing their markdown header from `###` to `####`. 

Also I think `jq` should have its own `####` subsection with some tips for installing directly from the readme, or linking to a guide like https://jqlang.github.io/jq/download/. (Just so it matches the other dependency installation steps)

Reply:
It makes sense. Let me fix it.

Reply:
I moved Foundry and Node to prerequisites. Workspaces (yarn install and yarn build) looks like part of day to day development process.

I can move Workspaces to prerequisites as well, if it does not look right.

@daniel-savu

Reply:
good spot, you're right 🫡